ampcrowd/basecrowd/urls.py

```diff
@@ -8,4 +8,6 @@
     url(r'^(\w+)/responses/$', views.post_response, name='post_response'),
     url(r'^(\w+)/tasks/$', views.create_task_group, name='create_tasks'),
     url(r'^(\w+)/purge_tasks/$', views.purge_tasks, name='purge_tasks'),
+    url(r'^(\w+)/summary/task_groups', views.get_task_groups, name='get_task_groups'),
+    url(r'^(\w+)/summary/tasks', views.get_tasks, name="get_tasks")
 )
```

Comment:
Yes, its dashboard. Would it be better to make that a different pull request, or include that? Those are basically poorly urled, json endpoints for tasks and task groups.